TestCases/compliance-level-3/0059-feel-all-function/0059-feel-all-function-test-01.xml

```diff
@@ -82,7 +82,7 @@
 
     <testCase id="009">
         <description>return null for singleton non-boolean</description>
-        <resultNode name="decision009" type="decision">
+        <resultNode errorResult="true" name="decision009" type="decision">
```

Comment:
Case:
https://github.com/dmn-tck/tck/blob/9f8ba50870259c971b5980d031fe73224635be09/TestCases/compliance-level-3/0059-feel-all-function/0059-feel-all-function.dmn#L82

DMN Spec:

> list of Boolean items or argument list of one or more Boolean items

Concur in the result, but I believe is missing the errorFlag to clarify this is an error case